sample.env

```diff
@@ -241,30 +228,31 @@ OBU_OTA_LOGGING_LEVEL="INFO"
 # Supported log levels are "ALL", "DEBUG", "ERROR", "FATAL", "INFO", "OFF", "TRACE" and "WARN"
 KC_LOGGING_LEVEL="INFO"
 
-# Set the MongoDB dataset to load, specified in the conflictmonitor/datasets directory
-CONFLICTMONITOR_MONGODB_INITIAL_DATASET="uturn_export"
-
 ##########################################
 # JPO ConflictMonitor Docker Environment #
 # Documentation: https://github.com/usdot-jpo-ode/jpo-conflictmonitor
 ##########################################
-
-# (Required) The IP address of Docker host machine which can be found by running "ifconfig"
-# Hint: look for "inet addr:" within "eth0" or "en0" for OSX
-# For Local deployments DB_HOST_IP and KAFKA_BROKER_IP should be set to DOCKER_HOST_IP, for remote deployments, these can be set independently.
 DB_HOST_IP=${DOCKER_HOST_IP}
 KAFKA_BROKER_IP=${DOCKER_HOST_IP}

```

Comment:
We have `ODE_KAFKA_BROKERS` set above, would this be different?

Reply:
the ODE_KAFKA_BROKERS is used by the jpo_count_metric in docker-compose-addons, while KAFKA_BROKER_IP is used by the conflictmonitor and conflictvisualizer_api. Let me check with Michael what that ODE_KAFKA_BROKERS variable does, and see if it is indeed the same

Reply:
This was not even used, removed